docs/_docs/02-ug-configuration.md

```diff
@@ -14,3 +14,5 @@ classes: wide
 | spark.hyperspace.explain.displayMode                 | "plaintext"                                                                                        | Display mode for Hyperspace explain() output. The valid set of values is: "console", "plaintext", "html".   | 0.1.0         |
 | spark.hyperspace.explain.displayMode.highlight.beginTag | "" (An empty string)                                                                                     | Tag to mark beginning of highlight portion in explain() output according to the display mode.         | 0.1.0         |
 | spark.hyperspace.explain.displayMode.highlight.endTag   | "" (An empty string)                                                                                    | Tag to mark ending of highlight portion in explain() output according to the display mode.            | 0.1.0         |
+| spark.hyperspace.index.lineage.enabled   | false                                                                                    | Add lineage column to index upon creation to track source data file for each index record.            | 0.3.0         |
```

Comment:
Can you also mention that this is used in refresh delete and the cons (storage?) of setting this to true?

Reply:
Sure, Updated it.